5001-6000/LIT5188MashafaMadbal.xml

```diff
@@ -1,20 +1,20 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5188MashafaMadbal" xml:lang="en" type="work">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ መድበል፡</title>
                 <title xml:lang="gez" corresp="#t1" type="normalized">Maṣḥafa Madbal</title>
-                
+
                 <title xml:lang="gez" xml:id="t2">ጉባዔ፡ ምሥጢራተ፡ ዘመናት፡</title>
                 <title xml:lang="gez" corresp="#t2" type="normalized">Gubāʿe Mǝśṭirata Zamanāt</title>
```

Comment:
Our guidelines only state the capitalising of the first word of titles, is there a particular reason for the capitalising throughout?

Reply:
no, I just have forgotten to check the titles